app/experimenter/nimbus-ui/src/components/PageNew/index.tsx

```diff
@@ -2,29 +2,70 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useCallback } from "react";
+import React, { useCallback, useState } from "react";
 import { RouteComponentProps } from "@reach/router";
 import AppLayout from "../AppLayout";
 import LinkExternal from "../LinkExternal";
 import FormExperimentOverviewPartial from "../FormExperimentOverviewPartial";
 
+import { useMutation } from "@apollo/client";
+import {
+  CREATE_EXPERIMENT_MUTATION,
+  CreateExperimentInput,
+  CreateExperimentResult,
+} from "../../models/Experiment";
+
 const TRAINING_DOC_URL =
   "https://mana.mozilla.org/wiki/display/FJT/Project+Nimbus";
 
 type PageNewProps = {} & RouteComponentProps;
 
 const PageNew = (props: PageNewProps) => {
-  // TODO: Get this from constants / config loaded at app start?
+  // TODO: EXP-462 Get this from constants / config loaded at app start?
   const applications = ["firefox-desktop", "fenix", "reference-browser"];
```

Comment:
No need to use a constant or load it at app start - since we can auto-generate types we can just grab this enum as demonstrated here: https://github.com/mozilla/experimenter/pull/3798/files#diff-978e704d3641f4527d638f00c3609cbbb5bd612bdd3b4ceb21eeeb271ea5c766R13-R17

(unless we feel like this needs to be super fresh and come directly from the server)

Reply:
Yeah, that's a concern for EXP-462 - been assuming that it'll need to come from the server, not sure how variable the application list will be

Reply:
We can still revisit in EXP-462, but I switched to the generated enum here

Reply:
Oh hrm, maybe these don't work here like I expected them to? I just tried to create an experiment and got `"FENIX" is not a valid choice.`

Reply:
Maybe they just need to be downcased. But I dont want this to block you; if we need to use the old const for now hthat's fine with me.